Add output of cekit run for Konflux

rdnovell opened the pull request:
This will add the output of Cekit run in dry-run mode.

Issue: https://issues.redhat.com/browse/CLOUD-4266
Signed-off-by: Ruben Novelli [rnovelli@redhat.com](mailto:rnovelli@redhat.com)

rdnovell committed:
Update PROVISIONING_MAVEN_PLUGIN_VERSION to 2.0.0.Beta1-redhat-00004

rdnovell committed:
Sync files /jdk17 builder-image

rdnovell committed:
Sync files /jdk21 builder-image

rdnovell committed:
Sync files /jdk17 runtime-image

rdnovell committed:
Sync files /jdk21 runtime-image

rdnovell commented:
/ok-to-test

Commit:
Update task-coverity-availability-check

Commit:
Fix sha

jfdenise commented:
@rdnovell , it seems that you removed the test folder in the builder image. This needs to be reverted. All the tests are in this repository.

Commit:
Remove test folders

rdnovell commented:
> @rdnovell , it seems that you removed the test folder in the builder image. This needs to be reverted. All the tests are in this repository.

Hi  @jfdenise I fixed the commit and removed only test from Modules folder
